crates/e2e/src/setup/services.rs

```diff
@@ -228,7 +228,11 @@ impl<'a> Services<'a> {
         );
         self.start_autopilot(
             Some(Duration::from_secs(11)),
-            vec!["--drivers=test_solver|http://localhost:11088/test_solver".to_string()],
+            vec![
+                "--drivers=test_solver|http://localhost:11088/test_solver".to_string(),
+                "--price-estimation-drivers=test_quoter|http://localhost:11088/test_quoter"
```

Comment:
This is the real change of the PR right? Where does `test_quoter` get registered and why is it different than `test_solver`?

Reply:
Well, it's the important part for allowing there to be a separate quoter endpoint. Inside of `start_protocol_legacy_solver` we create a `test_solver` and a `test_quoter`. The reason for this is because while solving and quoting look the same in terms of data model, solvers can have different endpoints for these tasks, we do at Enso.

Without this, quoting and solving will always go the same endpoint. With this change, it's more flexible and allows you to either:
a) provide a different endpoint for `test_quoter`
b) provide the same endpoint for `test_quoter`